pkg/monitor/prometheus/metrics.go

```diff
@@ -0,0 +1,87 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/apache/dubbo-admin/pkg/admin/config"
+	"github.com/apache/dubbo-admin/pkg/admin/constant"
+	"github.com/apache/dubbo-admin/pkg/admin/util"
+)
+
+func FetchRadio(metricName1 string, metricName2 string, label1 []string, label2 []string) (*QueryInfo, error) {
+	var query1 string
+	var query2 string
+	var query string
+	// Example: sum(http_success{}) / sum(http_total{})
+	var labelsQ1 string
+	for i, labelsInstance := range label1 {
+		if i == 0 {
+			labelsQ1 += labelsInstance
+		} else {
+			labelsQ1 += ", " + labelsInstance
+		}
+	}
+	query1 = fmt.Sprintf("sum(%s{%s})", metricName1, labelsQ1)
+	var labelsQ2 string
+	for i, labelsInstance := range label2 {
+		if i == 0 {
+			labelsQ2 += labelsInstance
+		} else {
+			labelsQ2 += ", " + labelsInstance
+		}
+	}
+	query2 = fmt.Sprintf("sum(%s{%s})", metricName2, labelsQ2)
+	query = fmt.Sprintf("%s/%s", query1, query2)
+	prometheusUrl := fmt.Sprintf("http://%s:%s", config.PrometheusIp, config.PrometheusPort)
+	ustr := prometheusUrl + constant.EpQuery + query
+	log.Println(ustr)
```

Comment:
Use Zap to log information, please refer to how the other parts of the project use zap.